lib/config.ts

```diff
@@ -274,9 +283,20 @@ export function get(name: string): any
                 convictConf.get('https.cert'))),
             ca: fs.readFileSync(path.resolve(__dirname,
                 convictConf.get('https.ca'))),
+            crl: fs.readFileSync(path.resolve(__dirname,
+                convictConf.get('https.crl'))),
             requestCert: true,
             rejectUnauthorized: true
         };
+
+        // Setup file watch for crl changes
+        fs.watch(path.resolve(__dirname, convictConf.get('https.crl')),
+                 (event: string, filename: string): void => {
+                    // Re-read the crl file
+                    otherConf['serverOptions'].httpsServerOptions.crl =
+                        fs.readFileSync(path.resolve(__dirname, convictConf.get('https.crl')));
+                    emitter.emit('change', 'https.crl');  // Signal the server that config changes
+                });
```

Comment:
Does 'path.resolve' need to be called twice (as an argument to 'watch' and in the callback)? Can't the 'filename' argument to the call back be used instead?